src/Faithlife.Utility/EnumerableUtility.cs

```diff
@@ -429,7 +429,7 @@ public static int SequenceHashCode<T>(this IEnumerable<T>? source, IEqualityComp
 
 			comparer ??= EqualityComparer<T>.Default;
 
-			return HashCodeUtility.CombineHashCodes(source.Select(x => comparer.GetHashCode(x)).ToArray());
+			return HashCodeUtility.CombineHashCodes(source.Select(x => comparer.GetHashCode(x!)).ToArray());
```

Comment:
Was going to say that we should start using [`HashCode.Combine`](https://docs.microsoft.com/en-us/dotnet/api/system.hashcode.combine?view=dotnet-plat-ext-5.0) when it's available, but it doesn't support using a particular comparer AFAICT.